docs/Manifest.toml

```diff
@@ -0,0 +1,305 @@
+# This file is machine-generated - editing it directly is not advised
+
```

Comment:
Not strictly necessary, but I tend to commit it for doc builds in my projects since it makes it less probably that the doc build will fail due to some new release of any of the packages here. Your call!

Reply:
Ok! If you believe it is a good practice, we can leave it here!